src/ripple/rpc/impl/ShardArchiveHandler.cpp

```diff
@@ -375,13 +375,17 @@ ShardArchiveHandler::next(std::lock_guard<std::mutex> const& l)
     // sleeps.
     auto const& url{archives_.begin()->second};
     auto wrapper = jobCounter_.wrap([this, url, dstDir](Job&) {
+        auto const ssl = url.scheme == "https";
```

Comment:
In my opinion the `auto const ssl` only obscures what you're up to.  My preference would be `bool const ssl`.  But do what you want...